locales-pending/premium.ftl

```diff
@@ -298,6 +298,29 @@ data-broker-profiles-estimated-time = Est. time to complete: { $estimated_time }
 # $exposure_reduction is the percentage of exposures that are data brokers.
 data-broker-profiles-exposure-reduction = Exposure reduction: { $exposure_reduction }%
 
+# Welcome to Premium Data Broker Profiles
+
+# Exposure reduction chart
+# The number inside <nr> will be displayed in a large font,
+# the label inside <label> will be shown in a smaller font.
+# Variables:
+#   $nr (number) - % of exposures reduced for the user
+exposure-reduction-chart-heading = <nr>{ $nr }</nr><percent>%</percent> <label1>exposures will</label1><label2>be reduced</label2>
```

Comment:
Sorry for the multiple comments, but I would even split this into 2 strings

```
exposure-reduction-chart-heading = <nr>{ $nr }</nr><percent>%</percent>
# This is displayed below a percentage, within a chart. Try to keep the translation as short as possible
exposure-reduction-chart-explanation = exposures will be reduced
```

Reply:
Yea I think I can split them into maybe 2-3 strings if that makes it easier @flodolo 
Most of the discussions of this will be in slack, so I'll mark this as resolved after I split the strings

Reply:
2 would be enough. If we split it into, we're forcing a structure into the sentence displayed below the number.

Reply:
Unfortunately, we use svg <text> for the chart, so it seems that each line needs to be specifically defined. Otherwise, the whole thing will need to be in one line

https://developer.mozilla.org/en-US/docs/Web/SVG/Element/text

Reply:
But split into 2 lines